### PR TITLE
PEP cleanup part 13

### PR DESCRIPTION
**Pull request by rowanseymour**
Final instalment (F841 is unused variables)

### CODE REVIEW

**Comment by norkans7** [comment]
:+1:

File: setup.cfg [hunk 1]
```diff
@@ -2,4 +2,4 @@
 max-line-length=120
 filename=*.py,*.py.dev
 exclude=./env/*,*/migrations/*,./config/*,./fab*
-ignore=E501,F811,F841
+ignore=E501
```
Comment (rowanseymour) [inline setup.cfg]: This is line length... which we'll never fix

File: temba/channels/tasks.py [hunk 2]
```diff
@@ -30,16 +30,14 @@ def send_msg_task():
     """
     Pops the next message off of our msg queue to send.
     """
-    logger = send_msg_task.get_logger()
-
     # pop off the next task
-    task = pop_task(SEND_MSG_TASK)
+    cur_task = pop_task(SEND_MSG_TASK)
```
Comment (rowanseymour) [inline temba/channels/tasks.py]: Was shadowing `task` decorator

File: temba/flows/models.py [hunk 3]
```diff
@@ -1439,12 +1438,12 @@ def start_call_flow(self, all_contact_ids, start_msg=None, extra=None, flow_star
         if not channel or CALL not in channel.role:
             return runs
 
-        (entry_actions, entry_rules) = (None, None)
-        if self.entry_type == Flow.ACTIONS_ENTRY:
-            entry_actions = ActionSet.objects.filter(uuid=self.entry_uuid).first()
-
-        elif self.entry_type == Flow.RULES_ENTRY:
-            entry_rules = RuleSet.objects.filter(uuid=self.entry_uuid).first()
+        # (entry_actions, entry_rules) = (None, None)
```
Comment (rowanseymour) [inline temba/flows/models.py]: Maybe this code can be removed, but was thinking someone put it here for reason...

File: temba/flows/tasks.py [hunk 4]
```diff
@@ -54,10 +54,8 @@ def export_flow_results_task(id):
 
 
 @task(track_started=True, name='start_flow_task')
-def start_flow_task(id):
```
Comment (rowanseymour) [inline temba/flows/tasks.py]: Shadows `id()`

File: temba/flows/views.py [hunk 5]
```diff
@@ -158,11 +159,6 @@ def render_to_response(self, context, **response_kwargs):
     class Choropleth(OrgPermsMixin, SmartReadView):
 
         def get_context_data(self, **kwargs):
-            from temba.values.models import Value
```
Comment (rowanseymour) [inline temba/flows/views.py]: Best not to use adhoc imports unless we really need them since they often lead to duplicated imports (like they did here)

**Pull request merged**